cnxpublishing/tests/test_subscribers.py

```diff
@@ -42,13 +42,75 @@ def test_startup_event(db_cursor, complex_book_one,
     assert ident_mapping[book_one.ident_hash] == payload['module_ident']
 
 
+@pytest.mark.usefixtures('complex_book_one', 'recipes')
```

Comment:
This is redundant by the way. You don't need to make it for use since these are in the parameters of the test function.

Reply:
Yeah, I was cargo-culting the def above that - now I see that's  for scope-pyramid-app, which defines no parameters. I'll fix. So, it's magic-strings, eh? Ahh, that explains the error message I got one when I used `cursor` instead of `db_cursor` in a def  - seems slightly surprising/non-python for a functions's non-keyword argument's string-value to mean anything outside the scope of the function, but hey, what ya gonna do?

Reply:
This is in the context of test anywhere, where the very name of the function itself is magic (`test_`) so, meh.

Reply:
any_way_